cmd/storj-sim/network.go

```diff
@@ -169,21 +170,18 @@ func newNetwork(flags *Flags) (*Processes, error) {
 
 	// Create gateways for each satellite
 	for i, satellite := range satellites {
-		accessKey, secretKey := randomKey(), randomKey()
 		process := processes.New(Info{
 			Name:       fmt.Sprintf("gateway/%d", i),
 			Executable: "gateway",
 			Directory:  filepath.Join(configDir, "gateway", fmt.Sprint(i)),
 			Address:    net.JoinHostPort(host, strconv.Itoa(gatewayPort+i)),
-			Extra: []string{
-				"ACCESS_KEY=" + accessKey,
-				"SECRET_KEY=" + secretKey,
-			},
+			Extra:      []string{},
 		})
 
 		// gateway must wait for the corresponding satellite to start up
 		process.WaitForStart(satellite)
 
+		accessKey, secretKey := randomKey(), randomKey()
```

Comment:
This won't be generated by `setup` command itself?

Reply:
ah, right, we can now get rid of this part